micropy/project/modules/stubs.py

```diff
@@ -67,17 +52,26 @@ def stubs(self) -> Union[StubManager, Sequence[DeviceStub]]:
             List[micropy.stubs.Stub]: List of stubs used in project.
 
         """
-        return self._resolve_subresource(self._stubs)
+        _stubs = self.context.get("stubs", [])
+        return self._resolve_subresource(_stubs)
 
-    @stubs.setter
-    def stubs(self, val):
-        """Sets project stubs.
+    def get_stub_tree(self, stubs) -> Sequence[Path]:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/app/BradenM/micropy-cli/pullRequest?prid=4594550)